plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSessionProperties.java

```diff
@@ -229,6 +230,11 @@ public IcebergSessionProperties(
                         // Session-level redirections configuration does not work well with views, as view body is analyzed in context
                         // of a session with properties stripped off. Thus, this property is more of a test-only, or at most POC usefulness.
                         true))
+                .add(doubleProperty(
+                        MINIMUM_ASSIGNED_SPLIT_WEIGHT,
+                        "Minimum assigned split weight",
+                        icebergConfig.getMinimumAssignedSplitWeight(),
+                        false))
```

Comment:
I don't think we should have a session property for this. Per my understanding, this is a safety toggle.
In particular,  a user may set value to 0.0 (or close to 0.0),  destabilizing  the cluster.

Reply:
(btw if you convince me this stays, it needs validator as in https://github.com/trinodb/trino/blob/a76ee407500a48146226ed345e0d2f86e822c060/plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveSessionProperties.java#L485 )

Reply:
i think we should remove this.
this pr https://github.com/trinodb/trino/pull/12656 removes this from hive

Reply:
@alexjo2144 we can go ahead with this PR, and i can update mine PR to remove from both